frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/NpmMojo.java

```diff
@@ -12,6 +12,7 @@
 import org.sonatype.plexus.build.incremental.BuildContext;
 
 import java.io.File;
+import java.util.ArrayList;
```

Comment:
It was being used to create an empty list. I've now changed it to use `Collections.emptyList` to be more consistent with the rest of the code.